src/Symfony/Component/Console/Descriptor/ReStructuredTextDescriptor.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Descriptor;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Helper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * ReStructuredTextDescriptor descriptor.
+ */
+class ReStructuredTextDescriptor extends Descriptor
+{
+    private $applicationTitleUnderlineChar = '#';
```

Comment:
Some context in case anyone needs it: RST format doesn't define which characters to use for `<h1>`, `<h2>`, `<h3>`, etc. RST defines a series of characters that you can use (`###`, `===`, `^^^`, `~~~`, etc.) and the first one found is considered `<h1>`, the second set of characters found is considered `<h2>`, etc.

-----

I'm not saying that we need to do the same here, but, for your consideration, the standard notation that we follow in Symfony Docs is:

```
h1 =====
h2 -----
h3 ~~~~~
h4 .....
```

Here we would be using:

```
h1 #####
h2 *****
h3 =====
h4 -----
```